src/stepfunctions/steps/utils.py

```diff
@@ -45,13 +49,19 @@ def get_aws_partition():
     return cur_partition
 
 
-"""
-ARN builder for task integration
-"""
-def resource_integration_arn_builder(service, api, integration_pattern=None):
+def get_service_integration_arn(service, api, integration_pattern=IntegrationPattern.RequestResponse):
+
+    """
+    ARN builder for task integration
+    Args:
+        service(str): name of the task resource service
+        api(enum): api to be integrated of the task resource service
```

Comment:
```suggestion
        api(str): api to be integrated of the task resource service
```

Reply:
Will replace "str" with "<Service>Api" as it could provide a clearer explanation

Reply:
That's more confusing as it references a type that doesn't exist. Keep as `str`. The parameter name is already `api`.

Reply:
So in the new commit I replaced it with "<Service>Api" which I think it is clearer than only "Api". Sorry forgot to update it here.
Let me know if that's still confusing. I can change it to "str". The reason I think "str" is not the best choice is: I implemented the API as one enum and fetch the value in the get_service_integration_arn method. "str" is not really reflecting the input.